### ISSUE TITLE
"--" is not allowed in comments in Epub format.

### ISSUE THREAD

**Issue by Ryan-DowlingSoka** [de3sw2aq1/ra-ebook-scraper#3]
"--" is not allowed in  comments in the Epub format. In the 25th chapter there is a -- inside of a comment. This causes services like Google Books to be unable to process the epub.

The easiest fix would be to strip out all comments from the scrape, or maybe inside of comments we could just replace "--" with "-".

I was able to fix this, not in code, but by searching through scrape.json and removing all comments with `<!--(.|[\r\n])*?-->` as my regex.

**Comment by de3sw2aq1** [comment]
Ah, yeah. I forgot about that.

I have an untested fix here:
https://github.com/de3sw2aq1/ra-ebook-scraper/tree/comment-fix

I added a `--convert-comments` flag to `./build.py`. I liked reading all of the comments when I discovered them in the HTML source. Without the flag they should be removed during building.

This is untested because of issue #2, currently I can't build it with the unicode symbols breaking things.
